plugins/modules/system/alternatives.py

```diff
@@ -41,6 +41,16 @@
       - The priority of the alternative.
     type: int
     default: 50
+  state:
+    description:
+      - C(present) - install the alternative (if not already installed), but do
+        not set it as the currently selected alternative for the group.
```

Comment:
Maybe to clarify:
```suggestion
        not set it as the currently selected alternative for the group. If it already
        is the selected alternative for the group, it will not change that.
```